generators/chipyard/src/main/scala/upf/UPFAspect.scala

```diff
@@ -0,0 +1,24 @@
+// See LICENSE for license details
```

Comment:
Can you add this comment about LICENSEs to all files in this PR (we should be better on this)

Reply:
Going to do this for all the UPF files since they are new, but will leave them out of files like build.sbt, Makefile, and others that were already created. Should I add it to them too?